mrb_config.rb

```diff
@@ -113,7 +113,12 @@
     l.library_paths << File.expand_path('soil', bvd)
     l.library_paths.uniq!
 
-    l.libraries << 'glfw'
+    unless Platform.darwin?
+      l.libraries << 'glfw' 
```

Comment:
glfw3 is the static library, there is an issue in the cmake file, causing it to build only the static library